src/services/risk/risks-around.js

```diff
@@ -0,0 +1,37 @@
+var Place = require('../../place.model');
+var isNumeric = require('is-numeric');
+
+module.exports = function(request, response) {
+  if (Object.keys(request.params).length === 0) {
+    return response.send(400, 'Invalid params.');
+  }
+
+  var radiusInKm = 1;
+  var kilometersPerDegree = 111.2;
+  var longitude = request.params.longitude;
+  var latitute = request.params.latitude;
+
+  if (!isNumeric(latitute) || !isNumeric(longitude)) {
+    return response.send(400, 'Invalid latitude or longitude.');
+  }
+
```

Comment:
Should be extract to a validation method

`
if (!isNumeric(latitute) || !isNumeric(longitude)) {
  return response.send(400, 'Invalid latitude or longitude.');
}
`